src/Binary/Loader/AbstractDoctrineLoader.php

```diff
@@ -22,16 +22,12 @@ abstract class AbstractDoctrineLoader implements LoaderInterface
      */
     protected $manager;
```

Comment:
kinda out of scope for this PR, but could we get rid of the LegacyObjectManager?